_data/banking.yml

```diff
@@ -98,6 +98,13 @@ websites:
       - hardware
     doc: https://www.bca.co.id/en/Individu/Produk/E-Banking/Klik-BCA
 
+  - name: Bank Leumi
+    url: https://www.leumi.co.il/
+    img: leumi.png
+    twitter: leumidigital
```

Comment:
From CircleCI: Twitter handle "leumidigital" should be "LeumiDigital".

Reply:
Done